captum/attr/_core/layer/layer_integrated_gradients.py

```diff
@@ -64,16 +72,22 @@ def __init__(self, forward_func, layer, device_ids=None):
 
     def attribute(
         self,
-        inputs,
-        baselines=None,
-        target=None,
-        additional_forward_args=None,
-        n_steps=50,
-        method="gausslegendre",
-        internal_batch_size=None,
-        return_convergence_delta=False,
-        attribute_to_layer_input=False,
-    ):
+        inputs: Union[Tensor, Tuple[Tensor, ...]],
+        baselines: Optional[
+            Union[Tensor, int, float, Tuple[Union[Tensor, int, float], ...]]
+        ] = None,
+        target: Optional[
+            Union[int, Tuple[int, ...], Tensor, List[Tuple[int, ...]]]
+        ] = None,
+        additional_forward_args: Any = None,
+        n_steps: int = 50,
+        method: str = "gausslegendre",
+        internal_batch_size: Optional[int] = None,
+        return_convergence_delta: bool = False,
+        attribute_to_layer_input: bool = False,
```

Comment:
Shouldn't this be optional too ? 
cc: @vivekmig

Reply:
I think this is fine the way it is, Optional[X] is equivalent to Union of X and None, in this case  attribute_to_layer_input will always take a bool value, even in the default case, and should never be None.